src/layers/core/grid-layer/grid-layer.js

```diff
@@ -59,6 +42,20 @@ mat4.lookAt(viewMatrixCompat, [0, 0, 0], [0, 0, -1], [0, 1, 0]);
 const viewMatrix = new Float32Array(viewMatrixCompat);
 
 export default class GridLayer extends Layer {
+  /**
```

Comment:
No big deal, but it looks a bit weird to have function param JSDoc floating like this, when we no longer have a constructor / params. 

We could potentially decide that we don't need these in the code, because all our props are documented separately in markdown, or make these as inline comments on the defaultProps "struct", or change the notation here.